napari/layers/image/image.py

```diff
@@ -197,7 +197,7 @@ def __init__(
         shear=None,
         translate=None,
         visible=True,
-    ) -> None:
+    ):
```

Comment:
Why is this None removed? Is it to "untype" the whole method so Mypy ignores it?